modules/ossm-rn-fixed-issues.adoc

```diff
@@ -16,62 +16,30 @@ Result - How has the behavior changed as a result?  Try to avoid “It is fixed
-
-* https://issues.jboss.org/browse/MAISTRA-79[MAISTRA-79] - Running the `istiooc cluster up` command results in the istio-operator namespace deploying a pod responsible for continually ensuring the Elasticsearch sysctl requirements are met. This loop runs constantly causing a significant load on the system running the cluster.
-
-* https://issues.jboss.org/browse/MAISTRA-110[MAISTRA-110] In large clusters, citadel can take a significant amount of time to create secrets. This may cause the installation to fail.
-
-* https://issues.jboss.org/browse/MAISTRA-157[MAISTRA-157] The conditional rate limiting feature does not restrict access as expected.
-
-* https://issues.jboss.org/browse/MAISTRA-196[MAISTRA-196] If you edit the installation to modify a parameter, for example to enable authentication, the new installation will fail due to the existence of the new 1.1 CRD configmaps.
-
-* https://issues.jboss.org/browse/MAISTRA-420[MAISTRA-420] [Multi-tenant implementation] The Jaeger agent pods are unscheduled as a result of port collision in multi-tenancy deployment on OpenShift 4.1.rc.5.
-
-* https://issues.jboss.org/browse/MAISTRA-245[MAISTRA-245] The sidecar injector pod fails to start if you are running the upstream community version.
-
-* https://issues.jboss.org/browse/MAISTRA-462[MAISTRA-462] [Multi-tenant implementation] After adding a namespace member to a second control plane, Kiali does not display the namespace member in the namespace list because the namespace for the second control plane is missing the `maistra.io/member-of` and `istio.openshift.io/member-of` labels. This is a result of the installation of the second control plane failing due to https://issues.jboss.org/browse/MAISTRA-464[MAISTRA-464](the operator can't create the `istio-ingressgateway` service in the second control plane, because the NodePort already being used by the first control plane). The workaround is to manually change the node ports of the `istio-ingressgateway` service in the first control plane's namespace (using `oc edit svc`). The operator will then be able to finish deploying the second control plane.
-
-* https://issues.jboss.org/browse/MAISTRA-466[MAISTRA-466] [Multi-tenant implementation] When you install more than one control plane, the operator overwrites the Kiali oauthclient yaml on any existing Kiali installations so that only the most recent installation functions. The Kiali oauthclient (`oc get oauthclients kiali`) contains a list of the authorized URLs that OpenShift OAuth will redirect to. If your URL is not part of this list, then the oauth login will fail and return the following message:
+* link:https://issues.jboss.org/browse/MAISTRA-357[MAISTRA-357] In OpenShift 4 Beta on AWS, it is not possible, out of the box, to access a TCP or HTTPS service through the ingress gateway on a port other than port 80. The AWS load balancer has a health check that verifies if port 80 on the service endpoint is active.
```

Comment:
_out of the box_ may not translate well; Perhaps _by default_?

Reply:
Fixed.